src/bundleList.js

```diff
@@ -127,6 +131,13 @@ function initBundleBuckets(roles) {
 function sortBundleByBuckets(roles, bundle, bundleBuckets) {
   roles.forEach((role) => {
     if (bundle.indexOf('-' + role + '-') !== -1) {
+
+      // skip invalid bundle names
+      if( !validBundleRegex.test( bundle ) ){
+        console.error( 'skip bundle', bundle );
```

Comment:
@orangejulius, It doesn't use the word error in the message. It just gets sent to stderr.

@missinglink, maybe just add `info:` to the message for clarity.